File: Projeto-TCC/ACS-Cadastro-Mobile/app/src/main/java/tcc/acs_cadastro_mobile/subModels/Nationality.java
package tcc.acs_cadastro_mobile.subModels;

import org.json.JSONException;
import org.json.JSONObject;

import java.io.Serializable;

import io.realm.RealmObject;

import static tcc.acs_cadastro_mobile.Constants.Citizen.CITY;
import static tcc.acs_cadastro_mobile.Constants.Citizen.NATIONALITY;
import static tcc.acs_cadastro_mobile.Constants.Citizen.NATION_BIRTH;
import static tcc.acs_cadastro_mobile.Constants.Citizen.UF;

public class Nationality extends RealmObject  implements Serializable {
    private String nationality, nationBirth, uf, city;

    public Nationality() {
        this("", "", "", "");
    }

    public Nationality(String nationality, String nationBirth, String uf, String city) {
        this.nationality = nationality;
        this.nationBirth = nationBirth;
        this.uf = uf;
        this.city = city;
    }

    public String getNationality() {
        return nationality;
    }

    public void setNationality(String nationality) {
        this.nationality = nationality;
    }

    public String getNationBirth() {
        return nationBirth;
    }

    public void setNationBirth(String nationBirth) {
        this.nationBirth = nationBirth;
    }

    public String getUf() {
        return uf;
    }

    public void setUf(String uf) {
        this.uf = uf;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public JSONObject asJson() throws JSONException {
        JSONObject json = new JSONObject();
        json.put(NATIONALITY.name(), nationality);
        json.put(NATION_BIRTH.name(), nationBirth);
        json.put(UF.name(), uf);
        json.put(CITY.name(), city);
        return json;
    }

}

File: Projeto-TCC/ACS-Cadastro-Web/subModels/Nationality.php
<?php

class Nationality
{
    const NATIONALITY = "NATIONALITY";
    const NATION_BIRTH = "NATION_BIRTH";
    const UF = "UF";
    const CITY = "CITY";

    private $nationality, $nationBirth, $uf, $city;

    function __construct($nationality, $nationBirth, $uf, $city)
    {
        $this->nationality = $nationality;
        $this->nationBirth = $nationBirth;
        $this->uf = $uf;
        $this->city = $city;
    }

    public function getValuesToDB()
    {
        return array(":" . self::NATIONALITY => $this->nationality,
            ":" . self::NATION_BIRTH => $this->nationBirth,
            ":" . self::UF => $this->uf, ":" . self::CITY => $this->city);
    }

    public static function getFromArray(array $array)
    {
        return new Nationality($array[self::NATIONALITY], $array[self::NATION_BIRTH], $array[self::UF], $array[self::CITY]);
    }
}